src/gluonts/dataset/split/splitter.py

```diff
@@ -36,13 +36,16 @@
 
     splitter = DateSplitter(
         prediction_length=24,
-        split_date=pd.Period('2018-01-31', freq='D')
+        split_date=pd.Period('2018-01-31', freq='D'), 
+        windows=7
     )
-    train, test = splitter.rolling_split(whole_dataset, windows=7)
+    train, test = splitter.split(whole_dataset)
```

Comment:
Just a reminder to also update docstrings (this one, but also those of classes) once you are done with the changes